release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -193,6 +193,24 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Bare Metal Hardware Provisioning*
+
+* Previously, Ironic failed to download an image for installation because it uses HTTPS by default and Ironic does not have the correct certificate bundle available. This issue is fixed by setting the image download as insecure to request a transfer without the certificate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953795[*BZ#1953795*])
+
+* Previously, when using dual stack deployments, worker node hostnames sometimes did not match the name inspected before deployment. This caused nodes to need manual approval. This has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955114[*BZ#1955114*])
+
+* Previously, in UEFI mode, the ironic-python-agent created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the ironic-python-agent configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966129[*BZ#1966129*])
```

Comment:
maybe `ironic-python-agent` in backticks both places) because it sounds like the name of a daemon or service.  See the [Quick reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) section of the docs guidelines.

Maybe "...after the agent downloads the RHCOS image."--to be crystal clear on what performs the action.